rpm/user-managerd.spec

```diff
@@ -0,0 +1,39 @@
+Name: user-managerd
+Version: 0.0.1
+Release: 1
+Summary: Sailfish user manager daemon
+Group: System/Base
+License: BSD-3-Clause
```

Comment:
Should be just BSD.
https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#SoftwareLicenses

Reply:
There's no 3-clause in that list but google found: https://spdx.org/licenses/BSD-3-Clause.html which does use this longer format for "short identifier". Anyway I copied this from some already existing package so I guess there's not yet convention for this :)

Reply:
We should use Fedora's convention which is to use the short names on that list. See https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_license_field